### DISCUSSION TITLE
Dynamic text color in Dashboard for better contrast

### DISCUSSION THREAD

**Discussion by DnlMcn** [ActivityWatch/activitywatch#1040] [Ideas]
Depending on the color that was set for an activity, it can become difficult to read the text drawn on top of it, since it's always white (at least in Dark mode).

As seen here:
![image](https://github.com/ActivityWatch/activitywatch/assets/76132040/17a42644-e64e-4809-b500-8f7e371397d4)

It would be great if the text became black if the color below it was bright enough.

**Comment by BelKed** [comment]
Duplicate of https://github.com/ActivityWatch/aw-webui/issues/412

**Reply by DnlMcn** [reply]
Oh, my bad. I guess I didn't look hard enough.
